showcase/types/template-registry.ts

```diff
@@ -104,4 +107,12 @@ export default interface ShowcaseTemplateRegistry {
   'mock/demo/breakpoints-visualizaton': typeof MockDemoBreakpointsVisualization;
   'Mock::Demo::BreakpointsRuler': typeof MockDemoBreakpointsRuler;
   'mock/demo/breakpoints-ruler': typeof MockDemoBreakpointsRuler;
+
+  // MOCK COMPONENTS
```

Comment:
```suggestion
  // These are supporting files for `form/super-select.hbs` which requires a component to be passed in for the showcase
```

Reply:
Yeah, the "mock" situation is a bit of a mess now. Let's circle back once the whole migration has ended, and we can think of a better/different naming convention for all these mock components.